project/SbtWhen.scala

```diff
@@ -0,0 +1,7 @@
+object SbtWhen {
+  implicit class When[A](a: A) {
```

Comment:
Shamelessly ~~stolen~~ inspired from https://stackoverflow.com/questions/7313604/conditional-invocation-of-a-method-in-scala...
I'm no Scala nor SBT expert, but any improvement is welcome.

Reply:
Haha, I'm also very good at SBT. Looks good if we actually need switching by condition for now.